BoostTestAdapter/Resources.resx

```diff
@@ -0,0 +1,298 @@
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="BadFileZone" xml:space="preserve">
+    <value>File {0} came from another computer and was blocked to help protect this computer.</value>
+    <comment>{0} represents a source file name</comment>
+  </data>
+  <data name="BatchException" xml:space="preserve">
+    <value>Exception caught while running test batch {0} [{1}] ({2})</value>
+    <comment>{0} represents a source file name
+{1} represents a list of test names
+{2} represents an exception message</comment>
+  </data>
+  <data name="BatchStrategyNotFound" xml:space="preserve">
+    <value>No valid test batching strategy was found. Tests skipped.</value>
+  </data>
+  <data name="BatchStrategyNotFoundFor" xml:space="preserve">
+    <value>No valid test batching strategy was found for {0}. Source skipped.</value>
+    <comment>{0} represents a source file name</comment>
+  </data>
+  <data name="CmdLineArguments" xml:space="preserve">
+    <value>CmdLine arguments: {0}</value>
+    <comment>{0} represents command line arguments</comment>
+  </data>
+  <data name="CouldNotCreateDbgHelp" xml:space="preserve">
+    <value>Could not create a DBGHELP instance for '{0}' to determine whether symbols are available.</value>
+    <comment>{0} represents a test executable file name</comment>
+  </data>
+  <data name="CouldNotLocateDebugSymbols" xml:space="preserve">
+    <value>Could not locate debug symbols for '{0}'. To make use of '--list_content' discovery, ensure that debug symbols are available or make use of '&lt;ForceListContent&gt;' via a .runsettings file.</value>
+    <comment>{0} represents a test executable file name</comment>
+  </data>
+  <data name="Debugging" xml:space="preserve">
+    <value>Debugging:   -&gt; [{0}]</value>
+    <comment>{0} represents a list of test names</comment>
+  </data>
+  <data name="DebuggingPropertiesNotFound" xml:space="preserve">
+    <value>Could not obtain debugging properties for {0}.</value>
+    <comment>{0} represents a source file name</comment>
+  </data>
+  <data name="Discovering" xml:space="preserve">
+    <value>Discovering ({0}):   -&gt; [{1}]</value>
+    <comment>{0} represents the name of a test discoverer class
+{1} represents a list of source file names</comment>
+  </data>
+  <data name="DiscoveryException" xml:space="preserve">
+    <value>Exception caught while discovering tests: {0} ({1})</value>
+    <comment>{0} represents an exception message
+{1} represents an exception HResult</comment>
+  </data>
+  <data name="DiscoveryExceptionFor" xml:space="preserve">
```

Comment:
Unused?

Reply:
Should have been used at ListContentDiscoverer.cs:129